contrib/update/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.7.2-alpine
+
+WORKDIR /serenata-update
+
+ADD requirements.txt requirements.txt
```

Comment:
Just a minor note, Docker recommends the `COPY` command instead of `ADD` in situations like that:

https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy

Reply:
Good to know ; ) Many thanks (fixed in e2b3ab05)